src/FormlyMapper/FormlyField/BirthdayField.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Linio\DynamicFormBundle\FormlyMapper\FormlyField;
+
+use Linio\DynamicFormBundle\FormlyMapper\FormlyField;
+
+class BirthdayField extends FormlyField
+{
+    const ORDER_ASC = 'asc';
+    const ORDER_DESC = 'desc';
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getFieldType()
+    {
+        return 'birthday';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function buildFieldTypeConfiguration(): void
+    {
+        $this->formlyFieldConfiguration['templateOptions']['type'] = $this->getFieldType();
+        $options = $this->fieldConfiguration['options'];
+
+        $order = isset($options['order']) && (strtolower($options['order']) == (self::ORDER_ASC || self::ORDER_DESC))
+        ? $options['order']
+        : self::ORDER_DESC;
+
+        unset($this->formlyFieldConfiguration['templateOptions']['order']);
+
+        if (!isset($options['minAgeAllowed']) || !isset($options['maxAgeAllowed'])
+            || !is_numeric($options['minAgeAllowed']) || !is_numeric($options['maxAgeAllowed'])
+            || $options['minAgeAllowed'] < 0 || $options['maxAgeAllowed'] < 0) {
+            $this->cleanAgeAllowedInTemplateOptionsArray();
+
+            return;
+        }
+
+        $minAgeAllowed = min($options['minAgeAllowed'], $options['maxAgeAllowed']);
```

Comment:
Why are we doing `min/max` here at all? Shouldn't we instead be validating that the max age is greater than min age, otherwise we should throw an invalid configuration exception? This seems to me like it would hide bad configurations which means the DX for this isn't great, as developers would only find this out after the application was deployed.

Reply:
Perfect! thanks for the advice, I will fix it :)